Composer/packages/client/src/pages/publish/Notifications.tsx

```diff
@@ -90,3 +90,43 @@ export const getPendingNotificationCardProps = (items: BotStatus[]): CardProps =
     ),
   };
 };
+
+export const getPendingQNANotificationCardProps = (): CardProps => {
+  return {
+    title: '',
+    description: formatMessage(`Creating QnA Maker`),
+    type: 'pending',
+    onRenderCardContent: (props) => (
+      <div css={cardContent}>
+        <Icon css={infoType} iconName="CloudUpload" />
+        <div css={cardDetail}>
+          <ProgressIndicator label={props.description} />
+        </div>
+      </div>
+    ),
+  };
+};
+
+export const getCompletedQNANotificationCardProps = (opts: { time: number }): CardProps => {
+  const statusIconStyle = css({
+    margin: '12px 0 0 -1px',
+    width: '12px',
+    height: '12px',
+    fontSize: '12px',
+    color: '#27AE60',
```

Comment:
can we get the closest matching color from FluentTheme.palette or Shared, Communication coors